src/msw/printdlg.cpp

```diff
@@ -76,6 +76,24 @@ class WinPrinter
         return result;
     }
 
+    wxMemoryBuffer GetData(int level) {
+        if (m_hPrinter)
+        {
+            DWORD buf_size = 0;
```

Comment:
A nitpick, but this should be `bufSize` to follow the usual naming conventions. Or just `size`.

Reply:
Fixed with my latest commit.